plugins/modules/cloud/misc/proxmox_disk.py

```diff
@@ -0,0 +1,726 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2022, Castor Sky (@castorsky) <csky57@gmail.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = r'''
+---
+module: proxmox_disk
+short_description: Management of a disk of a Qemu(KVM) VM in a Proxmox VE cluster.
+version_added: 5.5.0
+description:
+  - Allows you to perform some supported operations on a disk in Qemu(KVM) Virtual Machines in a Proxmox VE cluster.
+author: "Castor Sky (@castorsky) <csky57@gmail.com>"
+options:
+  name:
+    description:
+      - The unique name of the VM.
+      - You can specify either I(name) or I(vmid) or both of them.
+    type: str
+  vmid:
+    description:
+      - The unique ID of the VM.
+      - You can specify either I(vmid) or I(name) or both of them.
+    type: int
+  disk:
+    description:
+      - The disk key (C(unused[n]), C(ide[n]), C(sata[n]), C(scsi[n]) or C(virtio[n])) you want to operate on.
+      - Disk buses (IDE, SATA and so on) have fixed ranges of C(n) that accepted by Proxmox API.
+      - >
+        For IDE: 0-3;
+        for SCSI: 0-30;
+        for SATA: 0-5;
+        for VirtIO: 0-15;
+        for Unused: 0-255.
+    type: str
+    required: True
+  state:
+    description:
+      - Indicates desired state of the disk.
+      - >
+        I(state=present) intended only for creating new disk (with or without replacing existing disk,
+        see I(force_replace) option).
+      - For changing options in existing disk use I(state=updated). Options will be replaced, not appended.
+      - When I(state=detached) and disk is C(unused[n]) it will be left in same state (not removed).
+    type: str
+    choices: ['present', 'updated', 'grown', 'detached', 'moved', 'absent']
+    default: present
+  force_replace:
+    description:
+      - Force replace existing attached disk with the new one leaving old disk unused.
+      - When disk exists and I(force_replace=False) creation will be silently skipped.
+    type: bool
+    default: False
```

Comment:
```suggestion
    default: false
```
Always use `true`/`false` for YAML booleans.

Reply:
Fixed